### ISSUE TITLE
Defining the services we need

### ISSUE THREAD

**Issue by kapalex** [TechnionYP5777/SmartCity-Accessibility#250]
We need to talk about what exact services we want to expose, to be used by our gui, and how those services would work

Few services that come to mind
- The login service seems to be a bit complicated.  Obviously we would accept a username and a password, but because we are sending the info to the server, and the server actually logs in to the parse server, we need to think how to return some "cookie" that marks the user as being signed in.
Basically because the service does the actual log-in, it now happens on the server side and not on the client side, so we need to think how it should be implemented (Maybe do the actual log in on the client side, not with java but with js)
- Review service to get, create, delete reviews
- Location service to get, create, delete locations

@koralchapnik @yaelAmitay @ArthurSap @Kolikant

**Comment by Kolikant** [comment]
The searching (both of them) through query creation (so query creation)
Navigation
User profile as well

**Comment by yaelAmitay** [comment]
- I can search online about the login problem. although the naive solution is: user always send his name and password to the server. (not a good solution I'll look for a better one)
- I think you mentioned all the services we need but we can use this issue to decide on url.
for example, navigation could be something like that:
localhost\navigation?src=BLA&dst=BLA
I think that if we'll have it in one issue it'll be easy for us.

**Comment by yaelAmitay** [comment]
- about the login we are using tokens.
- do we still need this issue? does anybody else have idea if we forgot some services? if you do please open issues about them.
